document-storage-adapter/src/Pssg.DocumentStorageAdapter/Controllers/FileController.cs

```diff
@@ -106,6 +106,20 @@ public async Task<ActionResult> Download([FromBody] ViewModels.Download download
                         ? metaData[S3.METADATA_KEY_TAG3]
                         : String.Empty;
 
+                if (!String.IsNullOrEmpty(_configuration["CHECK_DOCUMENT_TYPE"]))
+                {
+                    // Check the file type is .tiff by filename
+
+                    if (fileName.Contains(".tiff"))
```

Comment:
please adjust this as follows:
1 - add code to force lowercase on filename
2 - change the comparison to .tif to catch both .tif and .tiff

Reply:
Fixed the issue to accept both .tif and .tiff types and pushed the changes.